pkg/runtime/runtime.go

```diff
@@ -573,6 +578,17 @@ func (a *DaprRuntime) initRuntime(opts *runtimeOpts) error {
 		a.appHealthChanged(apphealth.AppStatusHealthy)
 	}
 
+	// Enable dynamic loading in standalone mode if dynamic components directory is provided
+	if a.runtimeConfig.Mode == modes.StandaloneMode && a.runtimeConfig.Standalone.EnableDynamicLoading == true {
+		dir, err := os.Stat(a.runtimeConfig.Standalone.ComponentsPath)
+		if err != nil {
+			log.Fatalf("failed to get components directory: %s", err)
+		} else if !dir.IsDir() {
+			log.Fatalf("components path is not a directory: %s", a.runtimeConfig.Standalone.ComponentsPath)
+		} else {
+			a.watchPathForDynamicLoading()
```

Comment:
Suggestion - after components are loaded successfully we can start watching for dynamic loading - after a.loadComponents